tool_collections/samtools/samtools_stats/samtools_stats_options.py

```diff
@@ -0,0 +1,44 @@
+import pysam
+import os
+import tempfile
```

Comment:
This will go away completely. Instead I will use rely on BAM metadata, which now includes refs, lengths, RGs etc. thanks to a recent update by @blankenberg